src/main/java/org/cyclonedx/util/ExtensionSerializer.java

```diff
@@ -80,6 +80,7 @@ private void serializeVulnerabilities(final ToXmlGenerator gen, final Extension
       generateTextNode(staxWriter, Vulnerability10.DESCRIPTION, vuln.getDescription(), Vulnerability10.NAMESPACE_URI, Vulnerability10.PREFIX);
```

Comment:
minor: up around line 46: another generify tweak:

```patch
Index: src/main/java/org/cyclonedx/util/ExtensionSerializer.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/cyclonedx/util/ExtensionSerializer.java b/src/main/java/org/cyclonedx/util/ExtensionSerializer.java
--- a/src/main/java/org/cyclonedx/util/ExtensionSerializer.java	(revision a2fe9c4287ed6c77a602c804978c506d4b490082)
+++ b/src/main/java/org/cyclonedx/util/ExtensionSerializer.java	(date 1614193412698)
@@ -43,7 +43,7 @@
     this(null);
   }
 
-  public ExtensionSerializer(final Class t) {
+  public ExtensionSerializer(final Class<Extension> t) {
     super(t);
   }
 
```

Reply:
In: e0b54de